web/src/components/tasks/LicenseStatusReceipt.tsx

```diff
@@ -127,27 +124,25 @@ export const LicenseStatusReceipt = (props: Props): ReactElement => {
       return "";
     }
     const { nameAndAddress } = business.licenseData;
-
     const secondLineAddress = nameAndAddress.addressLine2 ? ` ${nameAndAddress.addressLine2}` : "";
-
     return `${nameAndAddress.addressLine1}${secondLineAddress}, ${nameAndAddress.zipCode} NJ`;
   };
 
-  const getIconColor = (item: LicenseStatusItem): string => {
-    if (props.status === "ACTIVE" || props.status === "PENDING") {
-      return item.status === "ACTIVE" ? activePermitTheme.iconColor : pendingPermitTheme.iconColor;
-    } else {
-      return item.status === "ACTIVE" ? grayPermitTheme.iconColor : pendingPermitTheme.iconColor;
-    }
-  };
-
-  const receiptItem = (item: LicenseStatusItem, index: number): ReactElement => {
-    const border = index === 0 ? "" : "border-top-1px border-base-light";
-
+  const receiptItem = (
+    item: LicenseStatusItem,
+    index: number,
+    receiptItems: LicenseStatusItem[]
+  ): ReactElement => {
+    console.log("ITEM", item);
     return (
-      <div className="padding-x-3 fdr fac width-100" data-testid={`item-${item.status}`} key={index}>
-        <Icon className={`${getIconColor(item)} usa-icon--size-3`}>{getIcon(item.status)}</Icon>
-        <span className={`margin-left-2 padding-y-1 fg1 receipt-item ${border}`}>{item.title}</span>
+      <div key={index} data-testid={`item-${item.status}`}>
+        <div className="flex flex-column fac tablet-flex-row width-full pt-1 tpt-0">
```

Comment:
We already have an H1 on this page, setting this to a L1 with create an accessability issue. I personally don't think it should have a heading level unless other areas of the page, like `permit status` have a heading level, you should check with design